### PR TITLE
feat(server): add address transactions api and tests

### PR DESCRIPTION
**Pull request by shaojunda**
1. add address transactions show api.
2. add address transactions show api test.
3. add faker gem.
4. remove message attribute from RequestErrorSerializer.

### CODE REVIEW

File: server/app/controllers/api/v1/address_transactions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+module Api
+  module V1
+    class AddressTransactionsController < ApplicationController
+      before_action :validate_query_params
+      before_action :validate_pagination_params, :pagination_params
+
+      def show
+        account = Account.find_by!(address_hash: params[:id])
```
[the remark this answers is not in the document]
Comment (shaojunda) [reply server/app/controllers/api/v1/address_transactions_controller.rb]: every address has its account.

[the remark this answers is not in the document]
Comment (classicalliu) [reply server/app/controllers/api/v1/address_transactions_controller.rb]: Why not call it `address` ?

[the remark this answers is not in the document]
Comment (shaojunda) [reply server/app/controllers/api/v1/address_transactions_controller.rb]: image
Account model mapping to page Address.

[the remark this answers is not in the document]
Comment (shaojunda) [reply server/app/controllers/api/v1/address_transactions_controller.rb]: Because I think the account is more accurate

**Pull request merged**